README.md

```diff
@@ -44,11 +44,18 @@ To bring up a dev environment, run `make docker-dev`.
 This brings up an additional vault service, used for integration testing of
 some auxiliary binaries.
 
-You can run the unit and integration tests via `make docker-test`
+### Testing
 
-Once you are in the Docker container you can run the web server via `go run bin/grant-server/main.go`.
```

Comment:
I don't think we actually need to run `go run /src/grant-server/main.go` like the documentation had expressed in the past. If so, I'll just remove 
```
`> go run /src/grant-server/main.go` # run this in the shell (Accurate? Do we need this?)
```

Reply:
sounds good